test/samples.js

```diff
@@ -15,7 +15,36 @@ test('Samples sould expose a fill method', (t) => {
   t.is(typeof len, 'number')
 })
 
-test('Samples sould expose a data getter', (t) => {
+test('Samples.read copies PCM data back out of a frame', (t) => {
+  using sourceFrame = makeAudioFrame()
+  const samplesA = new ffmpeg.Samples()
+  const len = samplesA.fill(sourceFrame)
+
+  const view = new Int16Array(
+    samplesA.data.buffer,
+    samplesA.data.byteOffset,
+    len / 2
+  )
+  for (let i = 0; i < view.length; i++) {
+    view[i] = (i * 13) % 32768
+  }
+
+  const samplesB = new ffmpeg.Samples()
+  using scratchFrame = makeAudioFrame()
+  samplesB.fill(scratchFrame)
+  samplesB.data.fill(0)
+
+  samplesB.read(sourceFrame)
+
+  t.is(samplesB.data.length, samplesA.data.length, 'same byte length')
+  for (let i = 0; i < view.length; i++) {
+    if (samplesB.data.readInt16LE(i * 2) !== samplesA.data.readInt16LE(i * 2)) {
```

Comment:
I think that `same-object` used by `brittle` seems to handle `Int16Array` https://github.com/holepunchto/same-object/blob/main/index.js#L58

Maybe we should test if that could work:

```suggestion
  t.alike(samplesB, samplesA)
  // or 
  t.alike(samplesB.data, samplesA.data)
```

Reply:
aaah! Heck yeah it does. Fixed.